lib.nix

```diff
@@ -2,37 +2,37 @@
 # nix-repl lib.nix
 
 let
-  hostPkgs = import <nixpkgs> {};
-  lib = hostPkgs.lib;
+  # Allow overriding pinned nixpkgs for debugging purposes via iohkops_pkgs
+  fetchNixPkgs = let try = builtins.tryEval <iohkops_pkgs>;
+    in if try.success
+    then builtins.trace "using host <iohkops_pkgs>" try.value
+    else import ./fetch-nixpkgs.nix;
+
+  pkgs = import fetchNixPkgs {};
+  lib = pkgs.lib;
 in lib // (rec {
-  # Allows to also generate the key compared to upstream genAttrs
-  genAttrs' = names: fkey: fname:
-    lib.listToAttrs (map (n: lib.nameValuePair (fkey n) (fname n)) names);
+  ## nodeElasticIP :: Node -> EIP
+  nodeElasticIP = node:
+    { name = "${node.name}-ip";
+      value = { inherit (node) region accessKeyId; };
+    };
 
-  # Given a list of integers and a function,
-  # generate an attribute set with that many nodes and call the function with node index
-  genNodes = nodeLimit: ids: f:
-             genAttrs' (lib.flip builtins.filter ids
-                          (id: id < nodeLimit))
-                       (i: "node${toString i}") f;
+  centralRegion = "eu-central-1";
+  centralZone   = "eu-central-1b";
```

Comment:
I'm not _really_ sure these definitions belong here..

Moved them here out of desperation, in attempt to pick apart the infamous `globals.nix`.